### ISSUE TITLE
Exposing Auto Properties In API(Suggestion)

### ISSUE THREAD

**Issue by VectorScalar** [chhoumann/MetaEdit#97]
It would be convenient if it was possible to add, edit and remove auto properties through the meta edit api rather than having to do everything through the user settings. That way property names as well as their values could be dynamically changed.

**Comment by shemraz** [comment]
I agree, this would be great! I wanna be able to populate the auto properties with a Dataview query
